integration-tests/build-result.js

```diff
@@ -1,11 +1,11 @@
-import test from "ava"
```

Comment:
This is me trying to make jest run test from this folder only. See https://github.com/facebook/jest/issues/1696

I'll rename it back